src/components/MenuSection/MenuSection.module.css

```diff
@@ -0,0 +1,79 @@
+.menuSection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  padding: 140px 120px;
+  background-image: url('src/assets/backgrounds/menu_bg.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+.title {
+  margin: 0;
+  text-align: center;
+  color: #35B8BE;
+  font-size: 50px;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  line-height: 55px;
+  letter-spacing: 1.65px;
+  word-wrap: break-word;
+}
+
+.description {
+  max-width: 440px;
```

Comment:
It's better not to define max-width. This width should be defined by parent component. Like Menu section is placed inside of the generic component section e.g. Or has padding, percentage of parent, nor max-width

Reply:
Removed max-width